docs/howtoguides/enable_ua_in_dockerfile.md

```diff
@@ -0,0 +1,123 @@
+# How to Enable Ubuntu Advantage Services in a Dockerfile
+
+Ubuntu Advantage (UA) comes with several services, some of which can be useful in docker. For example, Extended Security Maintenance of packages and FIPS certified packages may be desirable in a docker image. In this how-to-guide, we show how you can use the `ua` tool take advantage of these services in your Dockerfile.
+
+
+## Step 1: Create a UA Attach Config file
+
+An attach config file for ua is a yaml file that specifies some options when running `ua attach`. The file has two fields, `token` and `enable_services` and looks something like this:
+
+```yaml
+token: TOKEN
+enable_services:
+  - service1
+  - service2
+  - service3
+```
+
+The `token` field is required and must be set to your UA token that you can get from signing into [ubuntu.com/advantage](https://ubuntu.com/advantage).
+
+The `enable_services` field value is a list of UA service names. When it is set, then the services specified will be automatically enabled after attaching (read: authenticating and authorizing) with your UA token.
+
+Service names that you may be interested in enabling in your docker builds include:
```

Comment:
Should we include `cis, usg and cc-eal` or they are too specific that we should not advertise them here ?

Reply:
yeah I'm not sure they make sense in a docker container context :shrug: